web/src/screens/settings/Irc.tsx

```diff
@@ -644,11 +667,15 @@ export const Events = ({ network, channel }: EventsProps) => {
   }, [settings]);
 
   return (
-    <div className="dark:bg-gray-800 rounded-lg shadow-lg p-1.5">
+    <div className={`dark:bg-gray-800 rounded-lg shadow-lg p-2 ${isFullscreen ? "fixed top-0 left-0 w-screen h-screen z-50" : ""}`}>
```

Comment:
Could use the `classes` helper function which will clean this up a bit. Same on line `672` and `678`.